golang/testsuite/testsuite_impl/static_file_consts/static_file_consts.go

```diff
@@ -3,6 +3,9 @@ package static_file_consts
 import "github.com/kurtosis-tech/kurtosis-client/golang/services"
 
 const (
-	TestStaticFileID services.StaticFileID = "test-static-file"
-	TestStaticFileFilename = "test-static-file.txt"
+	TestStaticFile1ID      services.StaticFileID = "test-static-file1"
+	TestStaticFile2ID							 = "test-static-file2"
```

Comment:
Shouldn't `TestStaticFile2ID` also be the same type ?

Reply:
Ooo yes nice catch! I tried to get clever, since Go will sometimes allow you to omit the type (taking the type of the line above), but not in this case. Rule: don't be clever 😆